ledger-core/virtual/execute/execute.go

```diff
@@ -235,13 +235,14 @@ func (s *SMExecute) stepWaitObjectReady(ctx smachine.ExecutionContext) smachine.
 			panic(throw.NotImplemented())
 		}
 	} else if objectState != object.HasState {
-		panic(throw.E("no state on object after readyToWork", struct {
+		s.prepareExecutionError(throw.E("no state on object after readyToWork", struct {
```

Comment:
maybe better to say error - "object is not ready"

Reply:
"object is not ready" means it will be ready later. 
Can we guarantee here that "object does not exist" ?